pkg/providers/aws/provider_postgres.go

```diff
@@ -974,6 +974,16 @@ func (p *PostgresProvider) exposePostgresMetrics(ctx context.Context, cr *v1alph
 	} else {
 		resources.SetMetric(resources.DefaultPostgresAvailMetricName, genericLabels, 1)
 	}
+
+	// cloud watch only provides us with free storage space, we need to expose more metrics to allow for more accurate alerting
+	// as `predict_linear` is hard to predict on non-linear growth & results in false positives
+	// we should follow the approach AWS take to auto scaling, and alert when free storage space is less than 10%
+	if instance != nil && instance.MaxAllocatedStorage != nil && instance.AllocatedStorage != nil{
+		// convert max allocated storage to bytes and expose as a metric
+		resources.SetMetric(resources.DefaultPostgresMaxAllocatedStorageMetricName, genericLabels, float64(*instance.MaxAllocatedStorage)*1e+9)
```

Comment:
Just to double check `MaxAllocatedStorage` is in GiB, right?  Not GB. Making sure this calculation is accurate

Reply:
Yeah just changing it to GiB after confirming Gibibytes 

https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#DBInstance

Reply:
Is this correct then? Gibibyte is `1024^3` bytes, right? Looks like we're doing `1000^3`